### PR TITLE
docs(website):added a blog about building a kubernetes application

### PR DESCRIPTION
**Pull request by shipsing**
### What does this PR do?
Adds a blog about building a Kubernetes application using Podman Desktop
### Screenshot / video of UI

https://672d92dd826e45d65faee0b2--podman-desktop-pr.netlify.app/blog/2024/10/05/kubernetes-blog
### What issues does this PR fix or reference?
https://github.com/containers/podman-desktop-internal/issues/353

### How to test this PR?

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: website/blog/2024-10-05-kubernetes-blog.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+- Deploy the application to Kubernetes
+- Verify the running
+ service
+
+## Building a containerized application 
+
+You can build a containerized application that uses:
+- a frontend Python application container
+- a backend Redis server container 
+
```
Comment (feloy) [inline website/blog/2024-10-05-kubernetes-blog.md]: This makes me think you can ONLY make this kind of application.

I would instead write something like `In this article, you will build a containerized ...`

File: website/blog/2024-10-05-kubernetes-blog.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+- Set up a local Kubernetes cluster
+- Deploy the application to Kubernetes
+- Verify the running
+ service
+
+## Building a containerized application 
+
+You can build a containerized application that uses:
+- a frontend Python application container
+- a backend Redis server container
```
Comment (feloy) [inline website/blog/2024-10-05-kubernetes-blog.md]: Below, you are describing how to pull and start the Redis server first. Perhaps the 2 containers should be ordered differently here, with the Redis container first?

**Comment by shipsing** [comment]
The suggestions provided by Vale look false positive.

File: website/blog/2024-10-05-kubernetes-blog.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+4. Click **Create Pod**.
+  ![copying containers to a pod](img/building-a-kubernetes-application/copying-containers-to-a-pod.png)
+5. View the newly created pod on the Pods page.
+6. Click the name of the pod and then click the **Summary** tab to view its summary.
+  ![viewing pod details](img/building-a-kubernetes-application/viewing-pod-details.png)
+
+
+**_Alternative: Pod creation with Kubernetes YAML_**
+
+You can generate a Kubernetes manifest for any existing pod or container and use it to create a local Kubernetes YAML file. Then, you can customize that file and create a pod from it.
```
Comment (feloy) [inline website/blog/2024-10-05-kubernetes-blog.md]: In this section, it is not very clear that we will create a Podman Pod, and not a Kubernetes Pod (as we are introducing the concept of a Kubernetes YAML, it is confusing)

File: website/blog/2024-10-05-kubernetes-blog.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+Based on your preference, use the Kind or Minikube extension:
+1. [Install the extension](/docs/extensions/install) from the Extensions catalog.
+2. Create a Kubernetes cluster. See [Kind cluster creation](/docs/kind/installing-extension) or [Minikube cluster creation](/docs/minikube/installing-extension).
+
+Once a kubernetes cluster is created, you can view a running control plane node and a running Kubernetes service on the Kubernetes component page. The page also shows that the cluster is connected.
+
+## Deploying the application to Kubernetes
+
+You can deploy the pod or container to a Kubernetes cluster that has an active connection and and access it through a service.
```
Comment (feloy) [inline website/blog/2024-10-05-kubernetes-blog.md]:
```suggestion
You can deploy the pod or container to a Kubernetes cluster that has an active connection and access it through a service. 
```

Comment (feloy) [inline website/blog/2024-10-05-kubernetes-blog.md]:
```suggestion
You can deploy the pod to a Kubernetes cluster that has an active connection and and access it through a service. 
```

I would not say that we can deploy a container to Kubernetes, as it only possible through a Pod

File: website/blog/2024-10-05-kubernetes-blog.md (hunk 3 above)
Comment (shipsing) [reply website/blog/2024-10-05-kubernetes-blog.md]: I have modified the content a bit for better clarity. 
I thought of adding this section to make readers aware that we have two options available for pod creation to give a complete picture. But yes, if it still looks confusing in the middle of the workflow, we can remove this section.

File: website/blog/2024-10-05-kubernetes-blog.md (hunk 4 above)
Comment (shipsing) [reply website/blog/2024-10-05-kubernetes-blog.md]: A quick question for understanding: If we cannot deploy a container to kubernetes, then why we get this option of `Deploy to Kubernetes` in the ACTIONS menu of the Containers page?

Comment (feloy) [reply website/blog/2024-10-05-kubernetes-blog.md]: In the Containers page, when choosing "Deploy to Kubernetes", a Pod is created (containing the container to deploy).

So effectively, if we consider the Podman Desktop experience, we can deploy a container to Kubernetes. I would change the sentence to something like below to clarify:

`You can deploy a pod or a container (inside a pod) to a Kubernetes cluster that ...`

Comment (shipsing) [reply website/blog/2024-10-05-kubernetes-blog.md]: Thank you for the clarification, @feloy

**Comment by shipsing** [comment]
@feloy, Thank you for the suggestions. Could you please verify the PR and confirm?

**Review by feloy — APPROVED** [review]
LGTM
Thanks for this article

**Review by slemeur — APPROVED** [review]
Nice one !

**Pull request merged**